lib/virtual-fs/src/webc_volume_fs.rs

```diff
@@ -300,12 +302,14 @@ impl AsyncWrite for File {
     }
 }
 
-// HACK: timestamps are not present in webc v2, so we have to return
-// a stub modified time. previously we used to just return 0, but that
-// proved to cause problems with programs that interpret the value 0.
-// to circumvent this problem, we decided to return a non-zero value.
+// HACK: WebC v2 doesn't have timestamps, and WebC v3 files sometimes
+// have directories with a zero timestamp as well. Since some programs
+// interpret a zero timestamp as the absence of a value, we return
+// 1 second past epoch instead.
 fn get_modified(timestamps: Option<webc::Timestamps>) -> u64 {
-    timestamps.map(|t| t.modified()).unwrap_or(1)
+    let modified = timestamps.map(|t| t.modified()).unwrap_or_default();
+    // 1 billion nanoseconds = 1 second
```

Comment:
nit: Comment for why this is the max?

Reply:
It's not the max, it's the min. `final value = max(value, min value)`

Reply:
Oh, duh.